cmd/fyne_settings/settings/appearance.go

```diff
@@ -89,11 +89,11 @@ func (s *Settings) LoadAppearanceScreen(w fyne.Window) fyne.CanvasObject {
 	return container.NewBorder(box, bottom, nil, nil, s.preview)
 }
 
-func (s *Settings) chooseTheme(name string) {
-	if name == systemThemeName {
-		name = ""
+func (s *Settings) chooseThemeVariant(variant string) {
```

Comment:
Not sure I would have renamed the method as it wasn't wrong in the way that variable names were

Reply:
Yes, I was unsure about that change 😅, will revert that renaming :)

Reply:
done